internal/services/authorization/role_assignment_resource.go

```diff
@@ -136,17 +136,14 @@ func resourceArmRoleAssignment() *pluginsdk.Resource {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ForceNew:     true,
-				RequiredWith: []string{"condition_version"},
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
 			"condition_version": {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ForceNew:     true,
-				RequiredWith: []string{"condition"},
 				ValidateFunc: validation.StringInSlice([]string{
-					"1.0",
```

Comment:
Whilst `2.0` is defaulted/supported for new resources (although it appears that's been removed from the referenced document at this point) any existing Role Assignments provisioned using `1.0` that's managed by Terraform will fail if we remove this - as such can we add `1.0` back in here?

Reply:
Yeah I will add this back in now